packages/Symfony/tests/Rector/BinaryOp/ResponseStatusCodeRector/ResponseStatusCodeRectorTest.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Symfony\Tests\Rector\BinaryOp\ResponseStatusCodeRector;
+
+use Rector\Symfony\Rector\BinaryOp\ResponseStatusCodeRector;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+final class ResponseStatusCodeRectorTest extends AbstractRectorTestCase
+{
+    public function test(): void
+    {
+        $this->doTestFiles([
+            __DIR__ . '/Fixture/fixture.php.inc'
+        ]);
+    }
+
+    protected function getRectorClass(): string
+    {
+        return ResponseStatusCodeRector::class;
+    }
+}
```

Comment:
Could you add 1-2 more tests? E.g.:

- with the other side match 
- code that is not in the list, e.g. `1241`